query/src/main/java/org/infinispan/search/mapper/mapping/impl/IndexProperties.java

```diff
@@ -31,25 +28,21 @@ public void setProperty(String key, Object value) {
    }
 
    public void setProperties(Map<String, Object> properties) {
-      properties.entrySet().stream()
-            .forEach(property -> setProperty(property.getKey(), property.getValue()) );
+      properties.forEach(this::setProperty);
    }
 
    public ConfigurationPropertySource createPropertySource(ConfigurationPropertyChecker propertyChecker) {
       ConfigurationPropertySource basePropertySource =
             propertyChecker.wrap(ConfigurationPropertySource.fromMap(backendProperties))
-                  .withPrefix(BACKEND_PROPERTIES_PREFIX);
+                  .withPrefix(EngineSettings.BACKEND);
       ConfigurationPropertySource propertySource =
             basePropertySource.withOverride(ConfigurationPropertySource.fromMap(engineProperties));
       defaultProperties();
       return propertySource;
    }
 
    private void defaultProperties() {
-      engineProperties.put("hibernate.search.default_backend", INFINISPAN_BACKEND_NAME);
       backendProperties.put("type", "lucene");
       backendProperties.put("analysis.configurer", new DefaultAnalysisConfigurer());
-      backendProperties.put("thread_pool.size", "1");
-      backendProperties.put("index_defaults.indexing.queue_count", "1");
```

Comment:
Are you sure about this? I think we discussed this at some point and the Search 6 defaults weren't acceptable for Infinispan. I.e. for each index 10 indexing queues, and *for each cache* about as many threads as the number of processor cores available to the JVM on startup.

Note that having more indexing queues than threads is not useful.

Reply:
Noted, I revamped the config in https://github.com/infinispan/infinispan/pull/8815, and those will become Infinispan level properties. I was planning handling the defaults there.

Reply:
I have moved the default to the new config PR at #8815, thanks @yrodiere